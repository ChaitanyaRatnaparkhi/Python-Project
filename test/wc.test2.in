This    is  a   tab     seperated   file.