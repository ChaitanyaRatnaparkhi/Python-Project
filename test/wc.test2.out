        1       6       41      test\wc.test2.in